src/app/move-popup/dimMoveItemProperties.html

```diff
@@ -39,35 +39,37 @@
     </div>
   </div>
   <div class="item-xp-bar" ng-if="vm.item.percentComplete != null && !vm.item.complete" dim-percent-width="vm.item.percentComplete"></div>
-  <div ng-if="vm.item.reviewable" class="move-popup-tabs">
-    <span class="move-popup-tab" ng-class="{ selected: vm.tab==='default' }" ng-click="vm.tab='default'" ng-i18next="MovePopup.OverviewTab"></span>
-    <span class="move-popup-tab" ng-class="{ selected: vm.tab==='reviews' }" ng-click="vm.tab='reviews'" ng-i18next="MovePopup.ReviewsTab"></span>
-  </div>
-  <dim-item-review item="vm.item" ng-if="vm.item.lockable" ng-show="vm.tab === 'reviews'"></dim-item-review>
-  <div ng-show="vm.tab === 'default'">
-    <form ng-if="vm.item.taggable" name="notes"><textarea name="data" ng-i18next="[placeholder]Notes.Help" ng-maxlength="120" ng-model="vm.item.dimInfo.notes" ng-model-options="{ debounce: 250 }" ng-change="vm.updateNote()"></textarea></form>
-    <span class="textarea-error" ng-show="notes.data.$error.maxlength" ng-i18next="Notes.Error"></span>
-    <div class="item-description" ng-if="vm.itemDetails && vm.showDescription" ng-bind="::vm.item.description"></div>
-    <div class="item-details" ng-if="vm.item.classified" ng-i18next="ItemService.Classified2"></div>
-    <dim-item-stats item="vm.item" class="stats" ng-if="vm.itemDetails && vm.hasDetails"></dim-item-stats>
-    <div class="item-details item-perks" ng-if="vm.item.talentGrid && vm.itemDetails">
-      <dim-talent-grid talent-grid="vm.item.talentGrid" dim-infuse="vm.infuse(vm.item, $event)"></dim-talent-grid>
-    </div>
-    <div class="item-details item-sockets" ng-if="vm.item.sockets && vm.itemDetails">
-      <sockets sockets="vm.item.sockets" infuse="vm.infuse(vm.item, $event)"></sockets>
+  <div ng-if="vm.itemDetails" class="move-popup-details">
+    <div ng-if="vm.item.reviewable" class="move-popup-tabs">
```

Comment:
While there, also include a check to (I believe it's) $ctrl.canReview to only show the selectors if the "Show detailed weapon reviews and allow rating weapons" is checked and fix #2368

Reply:
I actually think the discoverability of detailed reviews was a good idea, so I'm keeping the options there and fixing the display issue.